src/build.rs

```diff
@@ -62,13 +63,30 @@ pub fn rustup_add_wasm_target(log: &Logger, step: &Step) -> Result<(), Error> {
 }
 
 /// Run `cargo build` targetting `wasm32-unknown-unknown`.
-pub fn cargo_build_wasm(log: &Logger, path: &Path, debug: bool, step: &Step) -> Result<(), Error> {
+pub fn cargo_build_wasm(
+    log: &Logger,
+    path: &Path,
+    profile: BuildProfile,
+    step: &Step,
+) -> Result<(), Error> {
     let msg = format!("{}Compiling to WASM...", emoji::CYCLONE);
     PBAR.step(step, &msg);
     let mut cmd = Command::new("cargo");
     cmd.current_dir(path).arg("build").arg("--lib");
-    if !debug {
-        cmd.arg("--release");
+    match profile {
+        BuildProfile::Profiling => {
+            // Force enable debug info, because builds that use the release
+            // cargo profile disables debug info.
+            cmd.env("RUSTFLAGS", "-g");
```

Comment:
I think we should not do this for the time being because it doesn't actually do anything useful in the short-term (nothing reads the dwarf info) and otherwise the names section is always present. 

I think eventually we'll want to do this, but for now it may be better to get cache hits for builds in release/profiling mode